diff --git a/src/post/index.ts b/src/post/index.ts
index c507127..32d4965 100644
--- a/src/post/index.ts
+++ b/src/post/index.ts
@@ -17,7 +17,7 @@ async function post(): Promise<void> {
     try {
         await post();
     } catch (error) {
-        console.error("Failed to run post step.");
-        return 1;
+        console.error("Failed to run post step.");
+        process.exit(1);
     }
 })();
